### DISCUSSION TITLE
Increment Not Working in Go Extension

### DISCUSSION THREAD

**Discussion by noob8boi** [pocketbase/pocketbase#5530] [Q&A]
`"slug-":  anyValue`
	
	
The code above is an example of a decrement modifier, but it doesn't seem to work in the Golang extension. Is this the wrong approach, or is it still in development? By the way, this worked in the Dart SDK.

**Comment by ganigeorgiev** [comment]
The above code sample is incomplete and I'm not sure what you are trying to do, nor what actually is the problem.

Try to elaborate more clearly what actually is the issue but treating the discussions and issues tracker as chatgpt is just lazy and low effort and I don't want to encourage this type of discussions anymore.

If you are trying to use field modifiers as part of the `Set` method then this will not work as it is not supported at the moment (support for field modifiers as part of the `Get`/`Set` methods is added in the [ongoing refactoring](https://github.com/pocketbase/pocketbase/discussions/4355)).

If you want to subtract from the existing value you can simply retrieve and transform the old one, for example:

```go
record.Set("credits", record.GetInt("credits") - something)
```

**Reply by noob8boi** [reply]
I just wanted to know the proper method of using field modifiers because they aren't documented. The code above was just the example of decrement modifier. Thanks for confirming it is still in refactoring phase. 

And I believe RunInTransaction() is better suited as an alternative. 

Eitherway, I will try to be more specific in future.

[Edit- edited the question for better understanding]
